Makefile

```diff
@@ -47,16 +47,20 @@ api:
 	       --openapi_out=fq_schema_naming=true,default_response=false:. \
 	       $(API_PROTO_FILES)
 
-java:
-	protoc --plugin=protoc-gen-grpc-java \
-    --grpc-java_out="$OUTPUT_FILE" \
-    --proto_path=./api \
-    --proto_path=./third_party \
-    --go_out=paths=source_relative:./api \
-    --go-http_out=paths=source_relative:./api \
-    --go-grpc_out=paths=source_relative:./api \
-    --openapi_out=fq_schema_naming=true,default_response=false:. \
-    $(API_PROTO_FILES)
+.PHONY: java_api
+# generate java_api proto
+# REQUIRED: protoc-gen-grpc-java plugin from https://repo.maven.apache.org/maven2/io/grpc/protoc-gen-grpc-java/1.63.0/
+# on your path. Then:
+#   chmod u+x <PATH>/protoc-gen-grpc-java-1.63.0-osx-aarch_64.exe
+#   ln -s <PATH>/protoc-gen-grpc-java-1.63.0-<ARCH>.exe protoc-gen-grpc-java
+java_api:
+	mkdir -p ./relations-client-java/src/main/java
```

Comment:
What happens if the directory exists already?

Reply:
[mkdir(1) - Linux man page](https://man7.org/linux/man-pages/man1/mkdir.1.html)
mkdir - make directories
`-p, --parents
              no error if existing, make parent directories as needed,
              with their file modes unaffected by any -m option.`

So if the direction already exists, it does nothing.

Reply:
Perfect!